src/OAuth2.ts

```diff
@@ -134,6 +135,6 @@ export class OAuth2 {
     }
 
     private makeRedirectURL(token: string): string {
-        return path.join(this.redirectPrefix, token, "authorize");
+        return url.resolve(this.redirectPrefix, `${token}/authorize`);
```

Comment:
Looks like `url.resolve` got deprecated and has "Stability: 0".
https://nodejs.org/api/url.html#url_url_resolve_from_to

As far as I see this is supposed to be the replacement.
https://developer.mozilla.org/en-US/docs/Web/API/URL_API
Didn't find what this translates to nor if this is supported in NodeJS, besides the documentation note to us this API instead.

Reply:
ah fuck

Reply:
Now using URL, tests should ensure this is doing the right thing.